apps/zotonic_core/rebar.config

```diff
@@ -44,7 +44,7 @@
     {filezcache, "1.1.0"},
     {yamerl, "0.8.1"},
 
-    {zotonic_stdlib, "~> 1.6.0"},
+    {zotonic_stdlib, "~> 1.6"},
```

Comment:
Was downloaded `1.7.0` - maybe make sense to set here version `1.7`.

Reply:
Yes we want 1.7 as the minimum version.